entrypoint.sh

```diff
@@ -467,9 +489,14 @@ function restart_driver() {
 
     ${load_pci_hyperv_intf} && exec_cmd "modprobe -d /host pci-hyperv-intf"
 
+    timestamp_print "Apply blacklisted mofed modules file to host (${OFED_BLACKLIST_MODULES_FILE})"
+    trap 'remove_ofed_modules_blacklist' EXIT
+    generate_ofed_modules_blacklist 
     ${UNLOAD_STORAGE_MODULES} && unload_storage_modules
 
     exec_cmd "/etc/init.d/openibd restart"
+    remove_ofed_modules_blacklist
```

Comment:
I think we should also call to `remove_ofed_modules_blacklist` in `exit_entryp()` to make sure it is cleaned

Reply:
Why do you think that file worn't be removed?

Reply:
`exit_entryp()` exits with error code if `exec_cmd` failed so this line won't be executed on a failure

Reply:
I understand, but for error we have the trap calll on line [L493](https://github.com/Mellanox/doca-driver-build/pull/8/files#diff-6f9d41d046756f0ddc2fcee0626bdb50100d12b88f293734eff742818e03efa2R493)

Reply:
I see it now, wasn't there when I reviewed.